src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -87,25 +87,25 @@ export default Object.freeze({
   useLighthouseFormsSearchLogic: 'new_va_forms_search',
   vaGlobalDowntimeNotification: 'va_global_downtime_notification',
   vaOnlineFacilitySelectionV22: 'vaOnlineFacilitySelectionV22',
```

Comment:
@jenniemc is this one VAOS as well?

Reply:
@timwright12 Looks like we do reference `vaOnlineFacilitySelectionV22` in our app. I'll make the change and push the update to this PR.

Reply:
thanks!